torch/nn/functional.py

```diff
@@ -3728,6 +3736,8 @@ def interpolate(input: Tensor, size: Optional[int] = None, scale_factor: Optiona
         return torch._C._nn.upsample_linear1d(input, output_size, align_corners, scale_factors)
     if input.dim() == 4 and mode == "bilinear":
         assert align_corners is not None
+        if antialias:
+            return torch._C._nn.upsample_bilinear2d_aa(input, output_size, align_corners, scale_factors)
```

Comment:
Found a nice workaround to deal with the JIT FC window [here](https://github.com/pytorch/pytorch/issues/67565#issuecomment-961999987).

TL;DR: Gate the call to the new op appropriately using `torch.jit.is_scripting()` to avoid JIT FC issues while still allowing the new kwarg to be used outside of JIT scripting.